editor/src/clj/editor/camera_editor.clj

```diff
@@ -129,14 +133,14 @@
 
 (def outline-shader (shader/make-shader ::outline-shader outline-vertex-shader outline-fragment-shader))
 
-(defn- gen-outline-vertex [x y z w cr cg cb]
-  [(/ x w) (/ y w) (/ z w) cr cg cb 1])
+(defmacro ^:private gen-outline-vertex [x y z w cr cg cb]
+  `[(/ ~x ~w) (/ ~y ~w) (/ ~z ~w) ~cr ~cg ~cb 1.0])
```

Comment:
Turning this into a macro avoids having to box the primitive arguments for the function call.